torch/_dynamo/utils.py

```diff
@@ -397,6 +397,9 @@ def clone_tensor(x):
 
 def clone_input(x):
     """copy while preserving strides"""
+    if isinstance(x, torch._subclasses.FakeTensor):
+        # No need to clone fake tensors?
+        return x
```

Comment:
We should understand why we are cloning fake tensors in the first place

Reply:
In particular, even if fake tensors have no data, you may still be obligated to clone them in case metadata mutation happens

Reply:
This was causing issues on the branch, but we can pull it, maybe. Let me check.